packages/camera/camera_avfoundation/ios/camera_avfoundation.podspec

```diff
@@ -13,9 +13,13 @@ A Flutter plugin to use the camera from your Flutter app.
   s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :http => 'https://github.com/flutter/packages/tree/main/packages/camera_avfoundation' }
   s.documentation_url = 'https://pub.dev/packages/camera_avfoundation'
-  s.source_files = 'camera_avfoundation/Sources/camera_avfoundation/**/*.{h,m}'
-  s.public_header_files = 'camera_avfoundation/Sources/camera_avfoundation/include/**/*.h'
-  s.module_map = 'camera_avfoundation/Sources/camera_avfoundation/include/CameraPlugin.modulemap'
+  s.source_files = 'camera_avfoundation/Sources/camera_avfoundation*/**/*.{h,m,swift}'
```

Comment:
can you add a comment here to explain why we need to handle 2 folders (the `camera_avfoundation*` part).